src/rf/js/src/routes.js

```diff
@@ -2,8 +2,12 @@
 
 var router = require('./router').router,
     HomeController = require('./home/controllers').HomeController,
-    UserController = require('./home/controllers').UserController;
+    UserController = require('./user/controllers').UserController;
```

Comment:
I think this could be avoided by changing your exports to:
```
 module.exports = HomeController;
```
and 
```
module.exports = UserController;
```